pallets/time-release/src/weights.rs

```diff
@@ -58,106 +57,126 @@ pub trait WeightInfo {
+	/// Storage: `ParachainSystem::ValidationData` (r:1 w:0)
+	/// Proof: `ParachainSystem::ValidationData` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `TimeRelease::ReleaseSchedules` (r:1 w:1)
+	/// Proof: `TimeRelease::ReleaseSchedules` (`max_values`: None, `max_size`: Some(1449), added: 3924, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Locks` (r:1 w:1)
+	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1299), added: 3774, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Freezes` (r:1 w:0)
+	/// Proof: `Balances::Freezes` (`max_values`: None, `max_size`: Some(49), added: 2524, mode: `MaxEncodedLen`)
 	/// The range of component `i` is `[1, 50]`.
 	fn claim(i: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `201`
 		//  Estimated: `13824`
-		// Minimum execution time: 31_295_000 picoseconds.
-		Weight::from_parts(32_243_496, 13824)
-			// Standard Error: 1_886
-			.saturating_add(Weight::from_parts(1_706, 0).saturating_mul(i.into()))
+		// Minimum execution time: 30_819_000 picoseconds.
+		Weight::from_parts(31_598_199, 13824)
+			// Standard Error: 2_509
+			.saturating_add(Weight::from_parts(15_964, 0).saturating_mul(i.into()))
 			.saturating_add(T::DbWeight::get().reads(4_u64))
 			.saturating_add(T::DbWeight::get().writes(2_u64))
 	}
-	/// Storage: System Account (r:1 w:0)
-	/// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
-	/// Storage: TimeRelease ReleaseSchedules (r:0 w:1)
-	/// Proof: TimeRelease ReleaseSchedules (max_values: None, max_size: Some(1449), added: 3924, mode: MaxEncodedLen)
+	/// Storage: `System::Account` (r:1 w:1)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Locks` (r:1 w:1)
+	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1299), added: 3774, mode: `MaxEncodedLen`)
+	/// Storage: `Balances::Freezes` (r:1 w:0)
+	/// Proof: `Balances::Freezes` (`max_values`: None, `max_size`: Some(49), added: 2524, mode: `MaxEncodedLen`)
+	/// Storage: `TimeRelease::ReleaseSchedules` (r:0 w:1)
+	/// Proof: `TimeRelease::ReleaseSchedules` (`max_values`: None, `max_size`: Some(1449), added: 3924, mode: `MaxEncodedLen`)
 	/// The range of component `i` is `[1, 50]`.
 	fn update_release_schedules(i: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `39`
-		//  Estimated: `12503`
-		// Minimum execution time: 11_098_000 picoseconds.
-		Weight::from_parts(12_017_508, 12503)
-			// Standard Error: 2_053
-			.saturating_add(Weight::from_parts(71_460, 0).saturating_mul(i.into()))
-			.saturating_add(T::DbWeight::get().reads(1_u64))
-			.saturating_add(T::DbWeight::get().writes(1_u64))
+		//  Estimated: `13674`
+		// Minimum execution time: 27_421_000 picoseconds.
+		Weight::from_parts(28_366_892, 13674)
```

Comment:
nit: Radical change of weights

Reply:
Resolving this one since its related to time-release weights.